example/calculator.cpp

```diff
@@ -54,7 +54,7 @@ int main(int argc, char** argv)
         std::cerr << argv[1] << " is not a number or is larger than an int\n";
         return 1;
     }
-    mp::EventLoop loop("mpcalculator", LogPrint);
+    mp::EventLoop loop("mpcalculator", {LogPrint});
```

Comment:
You also need to update the loop initializaion in https://github.com/bitcoin-core/libmultiprocess/blob/master/example/printer.cpp#L52